documentation/rfc/ptref_grammar.md

```diff
@@ -165,6 +165,15 @@ Examples:
 * `/v1/coverage/id/lines?filter=get network <- line.id=Metro14` returns the lines from the network of the line Metro14.
 * `/v1/coverage/id/physical_modes?filter=get connection <- line.id=Metro14` returns the physical modes in connection with the line Metro14.
 
+## Data Graph
+The data is structured as a graph, where related objects are connected. When converting a collection into another, we use the graph to define the shortest series of jointures that needs to be performed to get to the target collection.
+
+The graph is as follow (edges may have weight to avoid unwanted behavior):
```

Comment:
As this is a functional doc, all shortcuts must be removed from the graph. The user can't known how the shortcuts are generated.

Reply:
I don't know if it should be removed from functional doc (it's already a pretty technical need).
But shortcuts need to be documented for technical purpose I think (debug and maintenance).